README.md

```diff
@@ -287,6 +287,25 @@ The push token will change after the reinstall. Update the push token to receive
 The SDK is using [kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly](https://developer.apple.com/documentation/security/ksecattraccessibleafterfirstunlockthisdeviceonly) to [save factors and keypairs](https://github.com/twilio/twilio-verify-ios/blob/main/TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift#L63). According to Apple, 
 > Items with this attribute do not migrate to a new device. Thus, after restoring from a backup of a different device, these items will not be present.
 
+<a name='NotificationExtension'></a>
```

Comment:
Please add this to the `Table of Contents`

Reply:
Added, Thank you.